src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final int AMOUNT = 1;
+    private static int supply = 0;
+    private static int buy = 0;
+    private static int result = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file): Divide program logic
Large methods is very difficult to read and understand. So methods should be short, between 5-15 lines. And it’s important for you to remember that one method should be responsible for one task.

For example: “write to file” or “create report” - are a separate operations. So we should create separate methods for them.